scripts/system/create/edit.js

```diff
@@ -2884,4 +2889,21 @@ selectionDisplay.onSpaceModeChange = function(spaceMode) {
     propertiesTool.setSpaceMode(spaceMode);
 };
 
+function getExistingZoneList() {
+    var center = {"x": 0, "y": 0, "z": 0};
+    var existingZoneIDs = Entities.findEntitiesByType("Zone", center, 17000);
```

Comment:
Magic number, 17000. (Coding standard 3.5.1.1.)
Why 17000?
A variable in upper case should be declared, where the variable name describes what the number is. (For example, line 44: `var MAX_DEFAULT_ENTITY_LIST_HEIGHT = 942;`)

Reply:
Good point.
About using a Constant
And this number was even not accurate now that re-thing about it.
This is the radius to search from the center of the domain to get the list of all zone entities.
so the value can't be just a bit larger than 16000, but must be sqr 2 * 16000 to be able to reach the corners.
I'm gonna use this:
var ENTIRE_DOMAIN_SCAN_RADIUS = 22628;